docs/management/admin/endpoint-artifacts.asciidoc

```diff
@@ -1,4 +1,38 @@
 [[endpoint-artifacts]]
+[chapter]
 = Optimize {elastic-defend}
 
-This page is a placeholder for future documentation.
+If you encounter problems like incompatibilities with other antivirus software, too many false positive alerts, or excessive storage or CPU usage, you can optimize {elastic-defend} to mitigate these issues.
+
+Endpoint artifacts — such as trusted applications and event filters — and Endpoint exceptions let you modify the behavior and performance of _{elastic-endpoint}_, the component installed on each host that performs {elastic-defend}'s threat monitoring, prevention, and response actions.
+
+The following table explains the differences between several Endpoint artifacts and exceptions, and how to use them for specific purposes:
+
+[cols="2"]
+|===
+
+| <<trusted-apps-ov,Trusted applications>>
+a| *_Prevents {elastic-endpoint} from monitoring a process._* Use to avoid conflicts with other software, usually other antivirus or endpoint security applications.
+
+* Creates intentional blind spots in your security environment — use sparingly!
+* Does not monitor the application for threats, and does not create alerts for the application, even if it behaves like malware, ransomware, etc.
+* Does not generate events for the application except process events for visualizations.
```

Comment:
Someone on the dev side might answer this better, but I think a trusted app is about preventing event _generation_, not event _collection_. That's why it's such a blindspot: the event data just doesn't exist, because it's not getting created.